app/decorators/staff/time_slot_decorator.rb

```diff
@@ -122,6 +122,10 @@ def display_presenter
     object.session_presenter || object.presenter
   end
 
+  def presenters_with_bios
+    object.program_session.speakers.map { |speaker| [speaker.name, speaker.bio] }
```

Comment:
This is a good place to use `pluck` which is more performant and elegant: 

```
object.program_sessions.speakers.pluck(:name, :bio)
```

Reply:
Good stuff. Many thanks. Will start using when relevant.